api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestPipeline.java

```diff
@@ -1656,4 +1656,100 @@ public void onNewDataReceived(TensorsData data) {
             fail();
         }
     }
+
+    /**
+     * Run SNPE with inception model with given runtime.
+     */
+    private void runSNPEInception(String runtime) {
+        File model = APITestCommon.getSNPEModel();
+        String desc = "appsrc name=srcx ! " +
+                "other/tensor,dimension=(string)3:299:299:1,type=(string)float32,framerate=(fraction)0/1 ! " +
+                "tensor_filter framework=snpe model=" + model.getAbsolutePath() +
+                " custom=Runtime:" + runtime + " !" +
```

Comment:
need space? `" !"`

Reply:
Without the space the pipeline works, but I updated the pipeline to have the space. Thanks!